data/xml/schema.rnc

```diff
@@ -12,8 +12,11 @@ MarkupText = (text | b | i | url | fixed-case | tex-math )+
 first = element first { text }
 last = element last { text }
 affiliation = element affiliation { text }
+orcid = element orcid { text }
+google-scholar = element google-scholar{ text }
+semantic-scholar = element semantic-scholar { text }
 Variant = element variant { attribute script { xsd:string }, (first? & last) }
-Person = attribute id { xsd:NCName }?, (first? & last & Variant? & affiliation? )
+Person = attribute id { xsd:NCName }?, (first? & last & Variant? & affiliation? & orcid? & google-scholar? & semantic-scholar? )
```

Comment:
Just a note on this: While we should absolutely start adding ORCIDs etc., we need to be careful to not introduce inconsistencies in our data. What if a name appears several times with the same ID but different ORCIDs?

This is ultimately a danger of attaching this information to _name instances_ on papers vs. to the _person identities_ themselves.